doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3706,6 +3706,21 @@ NCSDK-6238: Socket API calls might hang when using Download client
 
   **Workaround:** Set :kconfig:option:`CONFIG_DOWNLOAD_CLIENT_RANGE_REQUESTS` with the :ref:`lib_download_client` library.
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+
+Download client might fail with error ``-EMSGSIZE`` when using TLS on nRF91 Series devices.
+  The nRF91 Series modem has a size limit of 2303 bytes for receiving TLS packages.
+  This limitation will result in the download client failing with error ``-EMSGSIZE`` when the HTTP package responded by the server is larger than the modem size limit.
+  The HTTP package includes both the HTTP header and payload.
+  Samples and applications that are affected include those that use the :ref:`lib_download_client` library to download files through HTTP, or those that use :ref:`lib_fota_download` with modem updates enabled.
+
+  **Affected platforms:** nRF9160 nRF9131 nRF9151 nRF9161
+
+  **Workaround:** Set the :c:member:`download_client_cfg.frag_size_override`. This enables HTTP range requests with a limited payload size given by the :c:member:`download_client_cfg.frag_size_override`.
+                  The fragment size must be set so that the total size of the HTTP header plus the payload must not exceed the modem limit of 2303 bytes.
+                  The HTTP header size is dependent on the server in use.
+                  If you are experiencing this issue on a deployed product, reducing the HTTP header size responded by the server can also resolve this issue.
```

Comment:
```suggestion
  **Workaround:** Set the :c:member:`download_client_cfg.frag_size_override` event. 
  This enables HTTP range requests with a limited payload size as set in the :c:member:`download_client_cfg.frag_size_override` event.
  The fragment size must be set so that the total size of the HTTP header plus the payload must not exceed the modem limit of 2303 bytes.
  The HTTP header size is dependent on the server in use.
  If you are experiencing this issue on a deployed product, reducing the HTTP header size responded by the server can also resolve this issue.
```

Reply:
This is not an event. Will use configuration instead.